certified-connectors/Projectplace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1190 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
```

Comment:
Title needs to be updated

Reply:
@ygavish Please refer [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#submission-requirements)